doc/releasenotes.html

```diff
@@ -55,7 +55,37 @@ <h2>**CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical
 
 -->
 
-<h1 id="2020-11-29-r1">Version 2020.11.29 Revision 1, SSC Version 252: February 25, 2020</h1>
+<h1 id="2020-11-29-r1">Version 2020.11.29 Revision 2, SSC Version 253: March 12, 2021</h1>
+
+<p>This is a maintenance update to <a href="#2020-11-29-r1">SAM 2020.11.29 Revision 1</a>.</p>
+
+<ul>
+<li>Show heat map of system power generated on Summary page of Results page. (<a href="https://github.com/NREL/SAM/pull/571">571</a>)</li>
+<li>Fix weather file downloads from Advanced NSRDB Download window to set "utc" parameter to false so files are returned in local time instead of UTC. (<a href="https://github.com/NREL/SAM/pull/609">609</a>)</li>
+<li>Add minute value to error messages about weather files to facilitate troubleshooting problems. (<a href="https://github.com/NREL/SSC/pull/563">563</a>)</li>
+<li>Change Detailed PV short circuit current output variable labels to make it clear current is string current rather than array current. (<a href="https://github.com/NREL/SSC/pull/572">572</a>)</li>
+<li>Show system power generated by default on Results page time series tables and graphs instead of blank tables and graphs. (<a href="https://github.com/NREL/SAM/pull/559">559</a>)</li>
+<li>Fix issue with Electricity Bill calculations that caused SAM to crash when time-of-use periods have different numbers of tiers. (<a href="https://github.com/NREL/SSC/pull/560">560</a>)</li>
+<li>Improve Stochastic simulations so that input samples and results are consistent and add description of distributions to Help. (<a href="https://github.com/NREL/SAM/pull/565">565</a>)</li>
+<li>For Wind power model, only display Uncertanties tab on Results page when Uncertainty inputs are enabled and improve Uncertainties input page to make it clearer that inputs are optional. (<a href="https://github.com/NREL/SAM/pull/581">581</a>)</li>
+<li>Updated Solar Resource File Converter macro for SolarAnywhere files to work with latest SolarAnywhere version and fix issues with time stamp conversions. (<a href="https://github.com/NREL/SAM/pull/601">601</a>)</li>
+<li>Battery model:
+<ul>
+<li>Reorganize battery inputs so that sizing parameters are all in one place and closer to the sizing inputs, and fix issue with sizing using DC and AC capacities. (<a href="https://github.com/NREL/SAM/pull/569">569</a>, <a href="https://github.com/NREL/SAM/pull/560">560</a>)</li>
+<li>Add battery sizing option so battery capacity can be specified in kW and h, or kW and kWh to facilitate parametric simulations on battery capacity. (<a href="https://github.com/NREL/SAM/pull/606">606</a>)</li>
+<li>Fix behind-the-meter battery dispatch issues to prevent accidental discharging of battery to grid. (<a href="https://github.com/NREL/SAM/pull/561">561</a>)</li>
```

Comment:
https://github.com/NREL/ssc/pull/568 should be linked here or on its own bullet.